Jellyfin.Api/Controllers/CollectionController.cs

```diff
@@ -54,6 +60,12 @@ public class CollectionController : BaseJellyfinApiController
             [FromQuery] bool isLocked = false)
         {
             var userId = User.GetUserId();
+            var user = _userManager.GetUserById(userId);
+
+            if (!user.HasPermission(PermissionKind.IsAdministrator) || !user.HasPermission(PermissionKind.EnableCollectionManagement))
```

Comment:
Shouldn't this be a policy?

Reply:
I guess that makes more sense here, will change and force push once done.